modules/_labs/menu/react/lib/MenuItem.tsx

```diff
@@ -15,7 +15,7 @@ export interface MenuItemProps extends React.LiHTMLAttributes<HTMLLIElement> {
   /**
    * The function called when the MenuItem is clicked. If the item is a child of the Menu component, this callback will be decorated with the onSelect and onClose Menu callbacks. This callback will not fire if the item is disabled (see below).
    */
-  onClick?: (event: React.SyntheticEvent) => void;
+  onClick?: (event: React.MouseEvent<HTMLLIElement>) => void;
```

Comment:
I'm not sure these interfaces should be specific to an `<li>`. I don't think there is anything inherently useful to an application developer on the `HTMLLIElement` interface. We may use different ARIA roles depending on the context the `MenuItem` is used.

I checked what special attributes/properties are available to the LI element:

```ts
type OmittedProps = keyof Omit<HTMLLIElement, keyof HTMLElement>;
// 'type' | 'value'
```

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/li#Attributes

The `value` property is a number that represents the number of an ordered list.
The `type` property is deprecated.